cips/cip-blob-verified-signer.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Support authored blobs
+description: Introduce a new blob type that can be submitted whereby the signer address is included and verified.
+author: Callum Waters (@cmwaters)
+discussions-to: https://forum.celestia.org/t/cip-blobs-with-verified-author
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-05-22
+---
+
+## Abstract
+
+Introduce a new v2 blob type that can be submitted with the author of the blob. Validators verify that the author is correct, simplifying the loop for rollups that adopt a fork-choice rule that enshrines a specific sequencer.
```

Comment:
nit: This is useful for rollups that don't enshrine a sequencer as well! In fact, we don't enshrine a particular sequencer in Sovereign SDK

Reply:
Can you explain that a little further. My mental model was that the nodes had a list of accepted "authors" of blobs. Or perhaps another way of putting it is If anyone can submit blobs then why do you need to verify the signer.

Reply:
We have a list of accepted authors, but it's dynamic (no particular enshrined sequencer), and we even accept blobs from people who aren't on the list under some circumstances (i.e. to enable recovery if all sequences are offline/malicious)

Reply:
The actual rules are a little bit involved, but the high level idea is that we accept...
- All blobs from registered sequencers (i.e. actors who are subject to slashing on the rollup)
- Up to a (small) constant number of additional blobs per celestia block from unregistered sequencers

Reply:
I see so it's somewhat a middle ground between complete permissionless rollups and enshrined (whitelisted). I'll update the abstract accordingly

Reply:
> Up to a (small) constant number of additional blobs per celestia block from unregistered sequencers

The constant is so small that effectively rollup should be completely permissionless.

Reply:
is this resolved?

Reply:
Yes this is resolved